syntheticsclientv2/common_models.go

```diff
@@ -28,17 +28,17 @@ type Networkconnection struct {
 }
 
 type Advancedsettings struct {
-	Authentication     `json:"authentication"`
+	Authentication     *Authentication  `json:"authentication,omitempty"`
```

Comment:
If we don't use a pointer here then `authentication{}` will be sent to the api which will now give a 4xx stating that a valid username and password are required. 

Using the pointer means we dont send the field at all and thus no error.

This will need to be adapted for in anything relying on the client's types unfortunately (TF Provider)
Thus this will be a breaking change. :(